pkgs/applications/printing/cups-browsed/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+, cups
+, libcupsfilters
+, libppd
+, glib
+, libresolv
+, withAvahi ? true
+, avahi
+, withLDAP ? true
+, openldap
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cups-browsed";
+  version = "2.0b2";
+
+  src = fetchFromGitHub {
+    owner = "OpenPrinting";
+    repo = "cups-browsed";
+    rev = "9204bf3d89d82a0227d343f20ea1c55493eb85eb";
+    hash = "sha256-mgi2seDLrznrt4Q6uiSueWK7kai2BvG58JRx/VRVsW4=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
+  buildInputs = [
+    cups
+    glib
+    libppd
+    libcupsfilters]
+    ++ lib.optionals withAvahi [ avahi ]
+    ++ lib.optionals withLDAP [ openldap ]
+    ++ lib.optionals stdenv.isDarwin [ libresolv ];
+
+  # From weechat:
+  # Fix '_res_9_init: undefined symbol' error
+  CFLAGS = lib.optionalString (stdenv.isDarwin) "-DBIND_8_COMPAT=1 -lresolv";
```

```suggestion
  CFLAGS = lib.optionalString stdenv.isDarwin "-DBIND_8_COMPAT=1 -lresolv";
```
those can be ommited

---

resolved in rebased commits